gateway-provider-security-pac4j/src/main/java/org/apache/knox/gateway/pac4j/filter/Pac4jDispatcherFilter.java

```diff
@@ -122,6 +122,12 @@ public class Pac4jDispatcherFilter implements Filter, SessionInvalidator {
 
   private static final String PAC4J_OIDC_TYPE = "oidc.type";
 
+  /* property for specifying pac4j cookies ttl */
+  public static final String PAC4J_COOKIE_MAX_AGE = "pac4j.cookie.max.age";
+
+  /* default value is same is KNOXSSO token ttl default */
+  public static final int PAC4J_COOKIE_MAX_AGE_DEFAULT = -1;
```

Comment:
This could - should - be `private` (not used anywhere outside of this class) and `String` instead of `int` (i.e. `"-1"`, to avoid unnecessary conversion below)

Reply:
Agreed! thanks for the review!

Reply:
@smolnar82 actually  `PAC4J_COOKIE_MAX_AGE` is used outside of this class in  `KnoxSessionStore` which needs this to be public. 
I did update the `int` to `String` thanks for pointing it out!

Reply:
@moresandeep - yes, I know the parameter name is used elsewhere, that's why I added a comment on the default value only 😊
Thanks for applying the changes so quickly.